components/DatePicker/DatePicker.jsx

```diff
@@ -124,6 +124,7 @@ export default class DatePicker extends React.Component<Props> {
       className: classnames(
         className,
         "react-datetime",
+        "multi-input-group-item",
```

Comment:
<img width="894" alt="phrg-nitro" src="https://user-images.githubusercontent.com/1904314/41413793-8a78924c-6fba-11e8-8cdb-8f79c8431993.png">

Reply:
@rafbgarcia I think you may not have to add that css class, check out line `130`. That css class is optionally added to the component based on the `multiInput` prop.

Try the following:

```jsx
<DatePicker multiInput />
```